server/app.js

```diff
@@ -1,5 +1,109 @@
 const express = require('express');
+
 const app = express();
+const fs = require('fs').promises;
+
+const dbPath = './data.json';
+app.use(express.json());
+
+app.get('/api/tickets/labels', async (req, res) => {
+  /* An endpiont to get all labels.
+    loops through all ticket's labels and returns an array containing each label once */
+  const data = await fs.readFile(dbPath);
+  try {
+    const json = JSON.parse(data);
+
+    const labels = [];
```

Comment:
```suggestion
    const labels = json.map(ticket => ticket.labels || []).flat().filter(label => label === l)
    const uniqueLabels = [...new Set(labels)];
```

Reply:
Nice, wasn't familiar with the 'new Set(iterator)' trick to filter all duplicates. 
Thank you @tomeryp